docs/output/transfers.md

```diff
@@ -44,8 +44,8 @@ try {
       currency: "USD"
     },
     facilitatorFee: {
-      value: 8, // $0.8
-      currency: "USD"
+      total: 8, // $0.8
+      totalDecimal: "0.8049"
```

Comment:
In these create transfer examples, only `total` or `totalDecimal` should be present.